modules/tinymce/src/themes/silver/main/ts/ui/dialog/Dialogs.ts

```diff
@@ -10,6 +10,7 @@ import Env from 'tinymce/core/api/Env';
 import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
 import * as HtmlSanitizer from '../core/HtmlSanitizer';
 import * as NavigableObject from '../general/NavigableObject';
+import { dialogFocusShiftedChannel } from '../window/DialogChannels';
```

Comment:
Scoped imports might be preferable here
```suggestion
import * as DialogChannels from '../window/DialogChannels';
```